pldoc/static/js/colors.js

```diff
@@ -0,0 +1,132 @@
+define([
+    'jquery'
+], function($) {
+    'use strict';
+
+    var $pane = $('#colors-preview .info-pane'),
+        $title = $($pane).find('.color-info-title'),
+        $desc = $($pane).find('.color-description'),
+        $back = $($pane).find('.color-combinations.background'),
+        $ref = $($pane).find('.color-reference'),
+        $rgb = $($pane).find('.color-rgb'),
+        $hex = $($pane).find('.color-hex');
+
+    var $button;
+
+    var colors;
+
+    var ColorHandler = {
+
+        init: function() {
+            ColorHandler.setupJSONColors();
+            ColorHandler.listenForClick();
+            ColorHandler.listenForKeypress();
+            ColorHandler.listenForClose();
+        },
+
+        setupJSONColors: function() {
+            $.ajax({
+                async: false,
+                url: '/public/js/colors.json',
+                complete: function(data) {
+                    colors = data.responseJSON;
+                }
+            });
+        },
+
+        listenForClick: function() {
+            var classes;
+
+            $('#colors-preview').on('click', 'button.swatch', function() {
+                $button = $(this);
+                classes = $($button).attr('class').split(/\s+/);
+
+                ColorHandler.showInformationWindow(classes, $button);
+            });
+        },
+
+        listenForKeypress: function() {
+            var key;
+
+            $(document).on('keydown', function(e) {
+                key = e.which;
+
+                if (key === 27) {
```

Comment:
yeah, you should be able to get it with `'edx-ui-toolkit/js/utils/constants.js'` in requirejs